### DISCUSSION TITLE
App local

### DISCUSSION THREAD

**Discussion by byt3ware** [community/community#138284] [New to GitHub] [answered]
Como comparto mi aplicación Local con otras personas fuera de mi Red?

**Comment by AngelArgumedo** [comment, marked as answer]
Puedes usar Ngrok

https://ngrok.com

lo descargas y lo ejecutas, una vez en la terminal solo te autenticas con el siguiente comando:
ngrok config add-authtoken <tu_token_autenticacion>

Y para compartir tu app solo ejecutas tu app localmente y mira el puerto donde esta corriendo para luego usar este comando en el ngrok
ngork http http://localhost:<aqui pones el puerto>/

y listo....
